tests/__tests__/linear/sync_balance.ava.ts

```diff
@@ -174,14 +174,14 @@ workspace.test('sync balance', async (test, { root, contract, alice, owner }) =>
     );
   }
 
-  // -- amount balance diff < 100 yN
+  // -- amount balance diff < 1 NEAR
   await owner.call(
     v2,
     'adjust_balance',
     {
       account_id: contract.accountId,
-      staked_delta: 99,
-      unstaked_delta: 99
+      staked_delta: "99",
```

Comment:
maybe use `ONE_NEAR.subn(1).toString(10)` here which matches the original design of the test